.zap/rules.conf

```diff
@@ -1,9 +1,14 @@
-# Mark the following rules as INFO
+# Mark the following rules as IGNORE
 
 # CloudFlare will block the metadata endpoint access
-90034	INFO	(Cloud Metadata Potentially Exposed - Active/release)
+90034	IGNORE	(Cloud Metadata Potentially Exposed - Active/release)
 
-10096	INFO	(Timestamp Disclosure - Passive/release)
-10063-1	INFO	(Permissions Policy Header Not Set - Passive/beta)
-10055-4	INFO	(CSP - Wildcard Directive)
-40039	INFO	(Web Cache Deception)
+# Not applicable to the cloud appliaction
+10096	IGNORE	(Timestamp Disclosure - Passive/release)
+
+# TODO
+10063	IGNORE	(Permissions Policy Header Not Set - Passive/beta)
+
+# The applicationInsights endpoint will be removed
+10055	IGNORE	(CSP - Wildcard Directive)
+40039	IGNORE	(Web Cache Deception)
```

Comment:
Is this also fore appinsights?

Reply:
o, put it under the wrong place/comment. Tested not an issue in our cloud environment.  Updated,